### ISSUE TITLE
Testing accuracy of the Digits trained model using pycaffe doesn't match with Digits report

### ISSUE THREAD

**Issue by RileyLee** [NVIDIA/DIGITS#163]
Hi, there. I'm trying to build a detector using Digits trained model. I downloaded the mean.binary, deploy.prototype and .caffemodel file of the last epoch and feed them into pycaffe. However, I'm not able to get the same high validation accuracy from net.forward_all function (99.6% from Digits, 89.6% from Python). Before the classification, I subtract the mean image from the input image. I didn't check any resizing option in this process. Has anyone done this before? Is there any step I forget? I really appreciate your help!!!

BTW, I used the trained model from pycaffe and repeat the same validation procedure. The accuracy of pycaffe trained model matched with those displayed in the trained interface.

Here's part of my python code:
```python
mean = np.load(PY_MEAN_FILE)
net = caffe.Net(MODEL_FILE, PRETRAINED,caffe.TEST)
caffe.set_mode_cpu()
Image = cv2.imread(Image_Name)
Image = Image.transpose(2,0,1)
Image = np.float_(Image) - np.float_(mean)
out = net.forward_all(data=np.asarray([Image]))
predicted_label = out['prob'][0].argmax(axis=0)
```

**Comment by lukeyeager** [comment]
Check out the [classification example](https://github.com/NVIDIA/DIGITS/tree/v2.0.0-preview/examples/classification) at `examples/classification`. The README explains limitations and ways to extend the example.

**Comment by lukeyeager** [comment]
@RileyLee, did you solve your problem? I'm closing this issue. Please re-open it if you have more questions.

**Comment by karimhasebou** [comment]
@lukeyeager what if i don't do any image resizing in digits, should i still expect a difference. My model gets 98%  accuracy on test set in digits and 94 in pycaffe
